src/forest.nml

```diff
@@ -1,37 +1,443 @@
 
 /* *** Begin Forest *** */
 
-spritelayout sprlay_forest_grow_1 {
-	ground		{sprite:GROUNDSPRITE_NORMAL;}
-	building	{sprite:2072;}
+spritelayout sprlay_forest_temp_pine1_growing{
+	ground	{sprite:3962;}
+	//top corner
+	building	{
+	sprite: (construction_state==0 ? 1660 : construction_state==1 ? 1661 : construction_state==2 ? 1662 : 1663);
```

Comment:
This is a strange ternary, but I guess it works. Needs some spaces though. 😉 

```suggestion
	sprite: (construction_state == 0 ? 1660 : construction_state == 1 ? 1661 : construction_state == 2 ? 1662 : 1663);
```

Reply:
I had tried using the switch for this. Switch was returning an integer, as it should have been, But the NMLC errored on this each time and was demanding that switch returned spriteset. Which defeated the purpose of using in-game graphics.
So I used the same approach you did in ITL Houses and it worked.